fabric-data-attachment-api-v1/src/main/java/net/fabricmc/fabric/impl/attachment/AttachmentRegistryImpl.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.attachment;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Supplier;
+
+import com.mojang.serialization.Codec;
+import org.jetbrains.annotations.Nullable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.fabric.api.attachment.v1.Attachment;
+import net.fabricmc.fabric.api.attachment.v1.AttachmentRegistry;
+
+public final class AttachmentRegistryImpl {
+	private static final Logger LOGGER = LoggerFactory.getLogger("fabric-api-data-attachment-api-v1");
```

Comment:
fabric-**api**-data-atachment-**api**-v1? :D

Reply:
Makes more sense, but this seems a bit inconsistent across FAPI. I'll change it though.

Reply:
I think this can be automated with gradle (and use smth like `"$module_name" like in FMJ), but that is out of scope of this PR